docs/integrations/channels/guide/index.mdx

```diff
@@ -87,7 +76,6 @@ Once created, channels show up in the product list within the control panel so t
 
 <Callout type="info">
   - You can find the app ID in the URL when editing the app in [DevTools](https://devtools.bigcommerce.com/). For more information, see [Finding an App's ID](/docs/integrations/apps/guide/id).
-  - `config_meta.app.id` is optional; however, if you're building an app that creates or manages a channel, we recommend including the app ID to ensure the user interface in the BigCommerce control panel works properly.
   - Select partners who are promoted in the Channel Manager must build an app, and include the app ID in the create channel request.
 </Callout>
```

Comment:
I think this full callout can be removed. The app is automatically associated with the channel on channel creation which means partners don't need to provide an app id :x: